electron/app/components/NumericFieldFilter.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from "react";
+import { useRecoilState, useRecoilValue } from "recoil";
+
+import * as atoms from "../recoil/atoms";
+import * as selectors from "../recoil/selectors";
+import { NamedRangeSlider } from "./RangeSlider";
+import { getSocket } from "../utils/socket";
+import { animated, useSpring } from "react-spring";
+import useMeasure from "react-use-measure";
+
+const makeFilter = (fieldName, range, includeNone, isDefaultRange) => {
+  let expr,
+    rangeExpr = null;
+  let fieldStr = `$${fieldName}`;
+  if (!isDefaultRange) {
+    rangeExpr = {
+      $and: [{ $gte: [fieldStr, range[0]] }, { $lte: [fieldStr, range[1]] }],
```

Comment:
I found this error in the server logs:
```
pymongo.errors.OperationFailure: unknown top level operator: $gte
```
Still trying to determine how to reproduce, and whether it's this or the other instance of `$gte` that's causing it.

Reply:
This happens when unchecking "Show no value" - still not sure which query is causing the issue, though

Reply:
Fixed! Thanks, https://github.com/voxel51/fiftyone/pull/514/commits/80659e48101184bd09cb03c410bd220f368b5649

Reply:
Confirmed, thanks